test/ProjectService.Data.UnitTests/GetProjectUserTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using LT.DigitalOffice.ProjectService.Data.Provider;
+using LT.DigitalOffice.ProjectService.Data.Provider.MsSql.Ef;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using LT.DigitalOffice.UnitTestKernel;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.ProjectService.Data.UnitTests
+{
+    class GetProjectUserTests
```

Comment:
We don't have tests for repo? why isn't it there?

Reply:
Renamed